code/game/objects/structures/watercloset.dm

```diff
@@ -25,32 +26,51 @@
 /obj/structure/hygiene/proc/unclog()
 	clogged = 0
 
+/obj/structure/hygiene/proc/attempt_unclog(var/obj/item/thing, var/mob/user)
+	if(clogged <= 0)
+		user.visible_message("<span class='notice'>\The [src] isn't clogged.</span>")
+		return
+	if(being_unclogged)
+		user.visible_message("<span class='notice'>\The [src] is already being unclogged.</span>")
```

Comment:
Since your action is being blocked, a warning is more appropriate.

```suggestion
		user.visible_message(SPAN_WARNING("\The [src] is already being unclogged!")
```